### PR TITLE
[7267] - Change training route page for ITT reform

### PR DESCRIPTION
**Pull request by kennyevil**
### Context

https://trello.com/c/4BYpIBjV/7267-add-the-new-what-type-of-training-are-they-doing-page

### Changes proposed in this pull request

Change type of training page for new trainees.

### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [ ] Do we need to send any updates to DQT as part of the work in this PR?
- [ ] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -91,6 +91,7 @@ features:
   funding_uploads: true
   register_api: true
   lead_partners: false
+  itt_reform: false
```
Comment (defong) [inline config/settings.yml:94]: Can you add a `true` for the `review.yml`
So we can all marvel at your awesome work

:pray:

**Review by defong — COMMENTED** [review]
Code looks fine, just wanna default to true for review app so we can see

**Review by defong — APPROVED** [review]
Looks good to me
image

**Comment by tomtrentham** [comment]
This doesn't look quite right to me 🤔 

image

**Comment by andrewscrivener** [comment]
**Tested on;**
https://register-pr-4398.test.teacherservices.cloud/

The new layout looks good :shipit: 

I assume these fixes are on another branch but wanted to note that;

* "Create a trainee record" button isn't showing when logged in as Administrator (Agatha Baker - (Unexpected behaviour) or Denise Theominis and Emma Smith as LP (Expected behaviour)
* New route isn't displayed for any of the personas - Annie Bell, Damian Campbell, Denise Theominis as AP or Emma Smith as AP.

**Comment by kennyevil** [comment]
Cheers Andrew. The new route is addressed in #4400 , which should hopefully be ready for a review soon.

**Pull request merged**